HISTORY.md

```diff
@@ -3,6 +3,13 @@
 ### Public API changes
 * Add new API GetUnixTime in Snapshot class which returns the unix time at which Snapshot is taken.
 
+### New Features
+* Add SingleDelete for DB in C API
```

Comment:
These should go in the `Public API changes` section.

Reply:
Changed